site/zenodo_rdm/sitemap/templates/zenodo_sitemap/sitemap.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9 http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd">
+  {%- for url in urlset %}
+    <url>
+        <loc>{{ url.loc }}</loc>
+        {%- if url.lastmod %}
+        <lastmod>{{ url.lastmod }}</lastmod>
+        {%- endif %}
+        {%- if url.changefreq %}
+        <changefreq>{{ url.changefreq }}</changefreq>
+        {%- endif %}
```

Comment:
I think we can't reliably calculate this, since records/communities could change by users at any moment/frequency.